packages/react-vega-lite/package.json

```diff
@@ -18,14 +18,18 @@
     "types"
   ],
   "dependencies": {
-    "react-vega": "^6.0.1"
+    "react-vega": "^6.0.1",
+    "vega-embed": "^4.0.0"
   },
   "peerDependencies": {
     "react": "^16.0.0",
     "vega-lite": "^3.1.0"
   },
+  "devDependencies": {
+    "vega-lite": "^3.1.0"
```

Comment:
not necessary because this is already in the `devDependencies` of root `package.json` so when you `yarn install`, this monorepo will have `vega-lite` for development